homework-submission/react-week4/src/components/Week2/Week2.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import TodoItems from './TodoItems/TodoItems';
+import AddItem from './AddItem/AddItem';
+import todoData from './todoData/todoData';
+import shortid from 'shortid';
+import './Week2.css';
+
+const Week2 = () => {
+  const [data, setData] = useState({
+    items: todoData,
+  });
+
+  const deleteItemFnc = id => {
```

Comment:
No need to call functions `...Fnc`, it's clear that they are functions :wink:

Reply:
Thanks  @ArcoMul for reviewing this homework, and I will consider your tips in my work.